cmd/cmd.go

```diff
@@ -26,15 +26,18 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
-type Cmd interface {
-	// The cli app within this cmd
-	App() *cli.App
-	// Adds options, parses flags and initialise
-	// exits on error
-	Init(opts ...Option) error
-	// Options set within this command
-	Options() Options
-}
+type (
```

Comment:
Can you leave this type definition on one line please

Reply:
Ok, I will change